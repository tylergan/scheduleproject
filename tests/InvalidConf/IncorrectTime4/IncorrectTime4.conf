on Tuesday at 123 run /bin/date