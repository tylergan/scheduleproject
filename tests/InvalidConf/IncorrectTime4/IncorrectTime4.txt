Testing that runner.py outputs the correct output to STDERR when there is:
    1) Incorrect time as it must be formatted in military time