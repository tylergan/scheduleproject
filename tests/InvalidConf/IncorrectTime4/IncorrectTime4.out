error in configuration: on Tuesday at 123 run /bin/date
